FrontEnd/src/components/ProfileView/ProfileMenu/ProfileMenuLogistics/ProfileMenuLogistics.jsx

```diff
@@ -0,0 +1,24 @@
+import classes from "./ProfileMenuLogistics.module.css";
+import {profileInfo} from "../ProfileText.jsx";
```

Comment:
Format

Reply:
> Formating

It's not clear what the problem is

Reply:
```suggestion
import { profileInfo } from "../ProfileText.jsx";
```
Spaces before between brackets. We created a story to add eslint to the project.